src/arrayMethodSort.js

```diff
@@ -4,8 +4,17 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    String(a) > String(b) ? 1 : -1) {
+    for (let index = 1; index < this.length; index++) {
+      for (let i = 0; i < this.length - index; i++) {
+        if (compareFunction(this[i], this[i + 1]) > 0) {
+          [this[i], this[i + 1]] = [this[i + 1], this[i]];
```

Comment:
You are using `this[i]` & `this[i + 1]`. You can create variables `current = this[i]` & `next = this[i + 1]`

Reply:
Good idea to create a variable for this.